PWGHF/TableProducer/candidateSelectorD0.cxx

```diff
@@ -241,19 +241,15 @@ struct HfCandidateSelectorD0 {
           pidTrackNegKaon == TrackSelectorPID::Status::PIDAccepted) {
         pidD0 = 1; // accept D0
       } else if (pidTrackPosPion == TrackSelectorPID::Status::PIDRejected ||
-                 pidTrackNegKaon == TrackSelectorPID::Status::PIDRejected ||
-                 pidTrackNegPion == TrackSelectorPID::Status::PIDAccepted ||
-                 pidTrackPosKaon == TrackSelectorPID::Status::PIDAccepted) {
+                 pidTrackNegKaon == TrackSelectorPID::Status::PIDRejected) {
         pidD0 = 0; // exclude D0
       }
 
       if (pidTrackNegPion == TrackSelectorPID::Status::PIDAccepted &&
           pidTrackPosKaon == TrackSelectorPID::Status::PIDAccepted) {
         pidD0bar = 1; // accept D0bar
       } else if (pidTrackPosPion == TrackSelectorPID::Status::PIDAccepted ||
-                 pidTrackNegKaon == TrackSelectorPID::Status::PIDAccepted ||
-                 pidTrackNegPion == TrackSelectorPID::Status::PIDRejected ||
-                 pidTrackPosKaon == TrackSelectorPID::Status::PIDRejected) {
+                 pidTrackNegKaon == TrackSelectorPID::Status::PIDAccepted) {
```

Comment:
@fgrosa, I'm sorry to jump in after the PR is merged. But shouldn't these lines be
```
} else if (pidTrackNegPion == TrackSelectorPID::Status::PIDRejected ||
           pidTrackPosKaon == TrackSelectorPID::Status::PIDRejected) {
```
to match the behaviour of the D0?
In this way, the code also matches your comment for this PR.

Reply:
You are right, thanks for spotting! Fixed here: https://github.com/AliceO2Group/O2Physics/pull/2625